pidcat.py

```diff
@@ -224,6 +226,24 @@ def parse_start_proc(line):
     return line_package, '', line_pid, line_uid, ''
   return None
 
+ps_command = base_adb_command + ['shell', 'ps']
+ps_pid = subprocess.Popen(ps_command, stdin=PIPE, stdout=PIPE, stderr=PIPE)
+while ps_pid.poll() is None:
+  try:
+    line = ps_pid.stdout.readline().decode('utf-8', 'replace').strip()
+  except KeyboardInterrupt:
+    break
```

Comment:
This should exit the script. Right now it just lets execution continue rather than exiting.